protocols/purple/purple.c

```diff
@@ -1133,7 +1158,12 @@ void prplcb_conv_new(PurpleConversation *conv)
 		/* libpurple brokenness: Whatever. Show that we join right away,
 		   there's no clear "This is you!" signaling in _add_users so
 		   don't even try. */
-		imcb_chat_add_buddy(gc, gc->ic->acc->user);
+		if (!set_getbool(&(ic->acc->set), "defer_joins")) {
```

Comment:
Note to self: Considering the comment above, I wonder if this should be default behavior, because yeah, this ugly behavior affects quite a few prpls.

It's mostly a question of whether the defer point is always reached reliably - would, say, a newly created channel with no other members skip the add users call? Some weird prpl that just don't do it, and add them with delay?